plugins/MauticCrmBundle/Integration/SugarcrmIntegration.php

```diff
@@ -295,11 +339,11 @@ public function getAvailableLeadFields($settings = [])
                                     foreach ($fields['fields'] as $fieldInfo) {
                                         if (isset($fieldInfo['name']) && empty($fieldInfo['readonly'])
                                             && (!in_array(
-                                                $fieldInfo['type'],
-                                                ['id', 'team_list', 'bool', 'link', 'relate']
-                                            )
-                                            ||
-                                            ($fieldInfo['type'] == 'id' && $fieldInfo['name'] == 'id')
+                                                    $fieldInfo['type'],
+                                                    ['id', 'team_list', 'bool', 'link', 'relate']
```

Comment:
Same here, @kuzmany could you remove `'bool',` that disable boolean field mapping please?